src/main/java/gov/nasa/cumulus/metadata/aggregator/UMMGranuleFile.java

```diff
@@ -420,29 +441,47 @@ private JSONObject exportSpatial() throws ParseException{
         JSONObject geometry = new JSONObject();
         JSONObject horizontalSpatialDomain = new JSONObject();
         Boolean foundOrbitalData = false;
+        boolean isoBBoxAlreadyProcessed = false;
         spatialExtent.put("HorizontalSpatialDomain", horizontalSpatialDomain);
 
         if (granule instanceof IsoGranule) {
-            String polygon = ((IsoGranule) granule).getPolygon();
-            if (polygon != "" && polygon != null) {
-                // Export Polygon
-                addPolygon(geometry, polygon);
+            /**
```

Comment:
These contents seem identical to `readFootprintOrbitBBox`

Reply:
there are 3 if blocks in the MetadataFilesToEcho.readFootprintOrbitBBox. but those 3 if blocks are using XPath to extract footprint, orbit and bbox data and put the extracted data into IsoGranule Pojo.

The code within UMMGranuleFile.exportSpatial() also has 3 if blocks but these 3 blocks are taking data from IsoGranule pojo to constract Spatial JSON tree.  Or maybe I misunderstood the comment?  Please feel free to advise.